scripts/devtool/main.go

```diff
@@ -113,12 +113,8 @@ func main() {
 		}
 		group.Go(func() error {
 			err := admin.Wait()
-			if err != nil {
-				yellow.Println(err)
-			}
-			// nolint:all //required
 			red.Println("ADMIN SERVER STOPPED")
-			return err
+			return multierr.Append(err, context.Canceled)
```

Comment:
Why do you need to add `context.Canceled` manually? It should usually only be returned from `ctx.Err()`, so feels like something underlying might be wrong

Reply:
If admin server crashes due to port not available, it doesn't return an error on exit. We want downstream services to exit if admin server crashes. Returning `context.Canceled` seemed like an easy `hack` to stop all services. Changed it to use `context.WithCancel` to send cancellation signal.